### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by David68326** [getlantern/lantern-forum-cn#465] [General]
@Mike777Khoo 
再发一次付款截图和邮箱截图给您，请尽快安排。谢谢！
![付款](https://github.com/getlantern/lantern-forum-cn/assets/157975130/13287bd5-9608-486f-aeb7-d27121139e6c)
![邮箱](https://github.com/getlantern/lantern-forum-cn/assets/157975130/bbb4ba70-b760-447a-94a1-0aff33bd9d4b)

**Comment by MaxMKMax** [comment]
@David68326 Hi David, 您好! 我上周已经帮您提交上去提醒支付部门了. 现在有截图就方便安排了. 但整个退款过程需要 14 天. 先通知您一下.

您放心, 我们这里会帮您弄好退款的. 请放心. 谢谢!

**Reply by David68326** [reply]
好的，那我就安静地等14天......非常感谢。

**Reply by MaxMKMax** [reply]
@David68326 您放心吧. 好好休息. 我会特别备注您的案件的. 希望您再回来用蓝灯! 随时欢迎回来成为我们的专业版用户!

**Reply by David68326** [reply]
Hi @Mike777Khoo ,
请问我的退款可以办理了吗？现在已经20多天了，烦请尽快处理一下，谢谢！
（蓝灯依旧无法正常使用，我是用了其它的VPN才能登录到这里。）

**Reply by David68326** [reply]
@Mike777Khoo 是这个帖子，您再看看，发了几天您都没回复，所以我又新写了个帖子@了你。

**Comment by David68326** [comment]
非常感谢您的帮助！



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月25日(周四) 凌晨4:18
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 申请退款 (Discussion #465)




 
@David68326 您放心吧. 好好休息. 我会特别备注您的案件的. 希望您再回来用蓝灯! 随时欢迎回来成为我们的专业版用户!

**Comment by David68326** [comment]
Hi,Mike,
请问我的退款可以办理了吗？现在已经20多天了，烦请尽快处理一下，谢谢！
我在论坛里发消息没有回复，所以直接发这个邮件给您了。



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月25日(周四) 凌晨4:18
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 申请退款 (Discussion #465)




 
@David68326 您放心吧. 好好休息. 我会特别备注您的案件的. 希望您再回来用蓝灯! 随时欢迎回来成为我们的专业版用户!
